environment.yml

```diff
@@ -4,8 +4,11 @@ channels:
   - defaults
 dependencies:
   - python=3.7
-  # Installing TJPcov in two steps to avoid making NaMaster installation to
-  # fail due to (https://github.com/LSSTDESC/NaMaster/issues/138)
-  - tjpcov
-  - tjpcov[full]
   - mpi4py
+  - pip
+  - numpy
```

Comment:
I had to add 
```
- wheel
- namaster
```

To get this to work on a brand new conda environment install.

Reply:
Added